api/api-doc/storage_service.json

```diff
@@ -881,6 +881,15 @@
                           "allowMultiple":false,
                           "type":"string",
                           "paramType":"query"
+                      },
+                      {
+                          "name":"scope",
+                          "description":"Stream phase scope",
```

Comment:
`Stream phase scope` seems too terse.
Can you please clarify the description and elaborate a bit more about what it does?